### ISSUE TITLE
Game request: Niizuma Lovely x Cation

### ISSUE THREAD

**Issue by marconii2002** [morkt/GARbro#67]
https://vndb.org/v20406 Can you add support for this game ?

**Comment by morkt** [comment]
try new pre-release build.

**Comment by marconii2002** [comment]
I tested and images in evimage.dat are out of order when filenames are fine. I downloaded Team Guishen rip of this game and images order is fine and filenames are orginal.

Here is example. 

Image ev103c that I extracted
![ev103c-my_rip](https://cloud.githubusercontent.com/assets/22653453/26285679/be1bc076-3e54-11e7-9722-f27532167042.png)

Image ev103c from Team Guishen rip.
![ev103c-tg_rip](https://cloud.githubusercontent.com/assets/22653453/26285698/efedf542-3e54-11e7-9253-a8ce442e79a6.png)
